### ISSUE TITLE
League of Legends =The phenomenon that the game window stops after finishing my Win10 pc game

### ISSUE THREAD

**Issue by oklokl** [safing/portmaster#826]
**Pre-Submit Checklist**:

- Check applicable sources for existing issues:
  - [Windows Known Issues](https://docs.safing.io/portmaster/install/windows#known-issues)
  - [Linux Known Issues](https://docs.safing.io/portmaster/install/linux#compatibility)
  - [Github Issues](https://github.com/safing/portmaster/issues?q=is%3Aissue+label%3Abug)

**What happened**:
When you close the game, the game window displays 'Not Responding'. The game window does not close normally. I want the game to close this window and go to the launcher, but I can't go.

I tried allowing both 'outbound and inbound' but the symptoms were the same

**What did you expect to happen?**:
I hope it works.
I used Avast Firewall Antivirus before.
It worked fine here.

**How did you reproduce it?**:
https://youtu.be/k9FMqXaAQkI 

**Debug Information**:
Microsoft Windows [Version 10.0.19044.1949]
Windows 10 64bit kr Insider
Avira free

safing Version 0.9.3 

Other info: My windows optimization and limit settings.
https://cafe.daum.net/candan/ASdB/419
https://cafe.daum.net/candan/ASdB/421
https://cafe.daum.net/candan/ASdB/427

https://drive.google.com/file/d/1rDUbOEKqzOVEVzLxYJ8ZU5RSmTTOpEee/view?usp=sharing
game dmp
[logs.zip](https://github.com/safing/portmaster/files/9482177/logs.zip)

[League of Legends.txt](https://github.com/safing/portmaster/files/9482345/League.of.Legends.txt)
Bug 'Not Responding' This process is the key issue.

https://drive.google.com/file/d/16_dGmj73OLTq45F0ZeKhEajaj81z72E_/view?usp=sharing
ProcessMonitor League.of.Legends.exe

Copy Debug 
[info log.txt](https://github.com/safing/portmaster/files/9482204/info.log.txt)

This is easiest to do in the preview mode.
where is this mod?
Is there a link to explain with an image?. I can't find it.

![5](https://user-images.githubusercontent.com/1571600/188265841-4fb0a9b8-8adb-4d29-b28a-4fcab3b90a48.png)
other error

**Comment by dhaavi** [comment]
Hey @oklokl, thanks for the elaborate report.

Dumb question, but since you don't explicitly mention it: Did you try to reproduce the issue without Portmaster running?

Portmaster seems to have accepted all connections and I don't know how it could possibly interfere here.

**Comment by oklokl** [comment]
In my opinion. After the game is over... Voting proceeds.(Game internal system) https://wormyeeee.tistory.com/255
![33](https://user-images.githubusercontent.com/1571600/190444070-5bb33b55-0d0f-4fc3-b749-1c17703cd7f1.png)


And the game program tried to connect to port 80.
When I blocked port 80 with my router, it showed the same symptoms.
I don't block port 80. I recently tried blocking port 80.
Something I recently discovered...


And I had the same symptoms even without any special settings.
And the program accepted everything, but same symptom
When the game is over, it connects to port 80.
Send a ping somewhere with icmp. game program settings.
dll file is created in the temp folder.

And if you uninstall or close the Portmaster program, the symptoms disappear.
The game program is normally closed.

Organize my scenario. My abstract guess.
In the process of changing the game program process.
This seems to be happening while blocking unexpected behavior.
lost log.
Certain actions may not be detected by the program. I can't seem to respond.
I'm not a developer, so I can't know the details.
